x-pack/test/functional/services/ml/api.ts

```diff
@@ -870,6 +870,21 @@ export function MachineLearningAPIProvider({ getService }: FtrProviderContext) {
       await this.waitForJobState(jobConfig.job_id, JOB_STATE.CLOSED);
     },
 
+    async createAndRunAnomalyDetectionJob(
```

Comment:
I'd like to avoid the code duplication here (it's just one changed parameter). My suggestion is to adjust the options for `createAndRunAnomalyDetectionLookbackJob` so it can be called with an end time for the datafeed. Here's my suggestion (the diff doesn't look ideal, let me know if it's unclear): https://gist.github.com/pheyos/5d417f043c31826c889f6e30a3164c42

Reply:
Sweet, thanks!  Fixed in 2301c76b618edb95fd3d675d62635cab002b9a1e